### DISCUSSION TITLE
When is next release scheduled?

### DISCUSSION THREAD

**Discussion by setyb** [openstyles/stylus#1571] [Q&A]
There are currently important merged, but unreleased, pull requests.  When is the next release scheduled?

**Comment by tophf** [comment]
There's no schedule and I don't see anything critical in the new commits, so we'll probably wait a little more. Meanwhile maybe you can install the [current dev version from github](https://github.com/openstyles/stylus/archive/refs/heads/master.zip) as an unpacked extension.

**Reply by setyb** [reply]
@tophf Really looking forward to [this commit](https://github.com/openstyles/stylus/commit/4a05cfc9bb9255c3aff638dd0ea4b35cf99e053a) being released.  That bug is troublesome.

Might try the dev build, but then I'll have to remember to keep checking back for the official release.

**Reply by tophf** [reply]
Instead of checking you can click the Watch button in our github repo and choose Custom -> **releases**.
